companies/eastbankclub.json

```diff
@@ -0,0 +1,23 @@
+{
+    "slug": "eastbankclub",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "health"
+    ],
+    "name": "East Bank Club The Fitness Factory GmbH",
+    "runs": [
+        "SUPERFIT"
+    ],
+    "address": "Postfach 410108\n12111 Berlin\nDeutschland",
+    "phone": "+49 30 688381768",
+    "fax": "+49 30 688381750",
+    "email": "member@superfit.club",
```

Comment:
Same with the email listed there. That way, we can also set the `suggested-transport-medium` to `email`.